substrate/frame/membership/src/benchmark.rs

```diff
@@ -0,0 +1,181 @@
+// This file is part of Substrate.
```

Comment:
We normally call this file `benchmarking.rs` by convention.

Reply:
Ack.
Updated to `benchmarking.rs`.